serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/brreg/BrregService.java

```diff
@@ -6,17 +6,19 @@
 import no.difi.meldingsutveksling.serviceregistry.model.BrregEnhet;
 import no.difi.meldingsutveksling.serviceregistry.model.EntityInfo;
 import no.difi.meldingsutveksling.serviceregistry.model.OrganizationInfo;
+import no.difi.meldingsutveksling.serviceregistry.util.SRRequestScope;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'no.difi.meldingsutveksling.serviceregistry.util.SRRequestScope'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)